exporter/otlp-http/README.md

```diff
@@ -68,15 +68,17 @@ For additional examples, see the [examples on github][examples-github].
 
 The collector exporter can be configured explicitly in code, or via environment variables as shown above. The configuration parameters, environment variables, and defaults are shown below.
 
-| Parameter           | Environment variable                         | Default                             |
-| ------------------- | -------------------------------------------- | ----------------------------------- |
-| `endpoint:`         | `OTEL_EXPORTER_OTLP_ENDPOINT`                | `"http://localhost:4318/v1/traces"` |
-| `certificate_file: `| `OTEL_EXPORTER_OTLP_CERTIFICATE`             |                                     |
-| `headers:`          | `OTEL_EXPORTER_OTLP_HEADERS`                 |                                     |
-| `compression:`      | `OTEL_EXPORTER_OTLP_COMPRESSION`             | `"gzip"`                            |
-| `timeout:`          | `OTEL_EXPORTER_OTLP_TIMEOUT`                 | `10`                                |
-| `ssl_verify_mode:`  | `OTEL_RUBY_EXPORTER_OTLP_SSL_VERIFY_PEER` or | `OpenSSL::SSL:VERIFY_PEER`          |
-|                     | `OTEL_RUBY_EXPORTER_OTLP_SSL_VERIFY_NONE`    |                                     |
+| Parameter                 | Environment variable                         | Default                             |
+|---------------------------|----------------------------------------------|-------------------------------------|
+| `endpoint:`               | `OTEL_EXPORTER_OTLP_ENDPOINT`                | `"http://localhost:4318/v1/traces"` |
+| `certificate_file:`       | `OTEL_EXPORTER_OTLP_CERTIFICATE`             |                                     |
+| `client_certificate_file` | `OTEL_EXPORTER_OTLP_CLIENT_CERTIFICATE`      |                                     |
+| `client_key_file`         | `OTEL_EXPORTER_OTLP_CLIENT_KEY`              |                                     |
+| `headers:`                | `OTEL_EXPORTER_OTLP_HEADERS`                 |                                     |
+| `compression:`            | `OTEL_EXPORTER_OTLP_COMPRESSION`             | `"gzip"`                            |
+| `timeout:`                | `OTEL_EXPORTER_OTLP_TIMEOUT`                 | `10`                                |
+| `ssl_verify_mode:`        | `OTEL_RUBY_EXPORTER_OTLP_SSL_VERIFY_PEER` or | `OpenSSL::SSL:VERIFY_PEER`          |
```

Comment:
is there a constraint that we would like to put on disabling peer verification when using mTLS?

Reply:
I've found it useful to still be able to disable host verification when testing as the certs don't always match the servers. As they're often just localhost or were determined by an overly dynamic DNS.

In our internal use case the certificate verification is _sometimes_ only actually useful from the server's perspective, which is actually our internal use case: a publicly exposed collector behind a reverse proxy that verifies the client using a certificate we know it must already have. Admittedly in that case we do know hostname matches but in our earlier iterations of the setup it didn't and caused problem with some of the OTEL SDKs for other languages.